configuration/drugs/drugs.csv

```diff
@@ -56,15 +58,18 @@ Uuid,Void/Retire,Name,Concept Drug,Concept Dosage Form,Strength,_version:1
 ,,Paracetamol 200mg Tablet,Cambodia_Paracetamol,Tablet,200mg,
 ,,Paracetamol 325mg Tablet,Cambodia_Paracetamol,Tablet,325mg,
 ,,Paracetamol 100mg Tablet,Cambodia_Paracetamol,Tablet,100mg,
-,,"Penicillin V 1,000,000IU Tablet",Cambodia_Penicillin V,Tablet,"1,000,000IU",
+,,"Penicillin V 1,000,000iu Tablet",Cambodia_Penicillin V,Tablet,"1,000,000iu",

```

Comment:
Same thing here. Renaming will create a new drug.
Modifying the FSN means modifying the drug identification key. With the new name, the Initializer will see it as a new drug and create it. Same as for Folic Acid 5mg, just find the UUID it has been assigned in production and input it in the UUID column.